src/drivers/test/diff/IrqlNotSaved.sarif

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Can you use this as a template instead of saving the same version of this for every query?

Reply:
These files are a way for us to keep track of "baseline debt".  The SARIF files checked in are the expected results for running our tests today.  If a change happens that results in a test result changing, we would save that diff to this file for that rule.  ATM they're redundant because we haven't had a regression like that yet.